content/docs/reference/fields.md

```diff
@@ -12,20 +12,21 @@ Although some fields have more properties here is a list of common ones that are
-| `ui.max`       | If `{ list: true }` can provide a maximin amount of items (_optional_)                                                                                                                  |
-| `ui.component` | Used for [custom field components](/docs/extending-tina/custom-field-components)                                                                                                        |
-| `ui.validate`  | Used for [custom field validation](/docs/extending-tina/validation/)                                                                                                                    |
+| Property       | Description                                                                                                                                                                                                                       |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`         | The name of the field                                                                                                                                                                                                             |
+| `type`         | The [type of the field](/docs/reference/types/) to be used                                                                                                                                                                        |
+| `label`        | A human friendly label that will be displayed to the user (_optional_, defaults to `name`)                                                                                                                                        |
+| `required`     | If `true`, the collection cannot be saved without this field present (_optional_, defaults to `false`)                                                                                                                            |
+| `isTitle`      | Denote a field as the title of a collection. [See below for more details](#istitle) (_optional_, defaults to `false`)                                                                                                             |
+| `isBody`       | If `true` this field will be used as the body of the document. [See below for more details](#isbody) (_optional_, defaults to `false`)                                                                                            |
+| `nameOverride` | An optional property to allow exporting a field with a special character, that wouldn't be supported by the `name` property. E.g: `{ name: 'custom_id', nameOverride: 'id' }` or `{ name: 'my_field', nameOverride: 'my-field' }` |
```

Comment:
Maybe you could add something about how that this property will be the one used to save the document to the Github or the filesystem.